src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -189,7 +189,10 @@ def build_summary_data(
                 ctx=ctx, project=project, referrer=Referrer.DAILY_SUMMARY_KEY_ERRORS.value
             )
             if key_errors:
-                project_ctx.key_errors = [(e["group_id"], e["count()"]) for e in key_errors]
+                if features.has("organizations:snql-join", project.organization):
+                    project_ctx.key_errors = [(e["e.group_id"], e["count()"]) for e in key_errors]
```

Comment:
When you join you're required to put an alias on all conditions, which results in the alias name being in the results.